html_link_parser.py

```
@@ -0,0 +1,454 @@
+"""
+HTML Link Parser
+
+Specialized HTML parsing utilities for extracting and categorizing links from the
+A Lo Cubano Boulder Fest website's HTML files.
+
+This module provides comprehensive link extraction with categorization, metadata capture,
+and analysis capabilities specifically designed for the project's HTML structure.
+"""
+
+import os
+import re
```

Ruff (0.12.2)

`re` imported but unused

Remove unused import: `re`

(F401)